koregraph/api/spectogram.py

```diff
@@ -0,0 +1,34 @@
+import os
+import librosa
+import numpy as np
+import matplotlib.pyplot as plt
+from pathlib import Path
+
+
+def save_images_log_power_spectogram(file_path: Path):
+    '''Save log power spectogram image of the audio file
+
+    Args:
+        file_path (Path) : path to the audio file
+
+    Returns:
+        Path: path to the saved image
+    '''
+
+    image_folder = os.environ.get("IMAGE_FOLDER")
+    if not os.path.exists(image_folder):
+        os.makedirs(image_folder)
```

Comment:
- [ ] Extract to global environment variable
- [ ] Use `pathlib.Path` `mkdir(parents=True, exist_ok=True)` to check for existance